### ISSUE TITLE
Whitelisting Multiple IP Addresses

### ISSUE THREAD

**Issue by zachgitt** [coinbase/gdax-node#152]
Please add functionality to allow multiple IP address whitelistings for an API key.

**Comment by maxgitt** [comment]
What he means is to please add functionality to allow whitelisting of multiple IP addresses for an API key.

**Comment by AugustasL** [comment]
Having the same problem here.

**Comment by fb55** [comment]
Right now you’ll have to create one API key per IP address.

**Comment by zachgitt** [comment]
Binance, Bittrex, Poloniex provide this functionality. A simple comma delimited list of IPs would work. Supporting the dev community’s needs is important in building a robust API.

**Comment by rmm5t** [comment]
@zachgitt Agreed, but unfortunately, the gdax-node project doesn't have any direct control of this. Maybe @sezu has some say in this at GDAX directly. I know it's a bit of a dumpster fire at the moment, but a [direct support ticket to Coinbase](https://support.coinbase.com/) might be a better way to get them to prioritize your desired feature.

> A simple comma delimited list of IPs would work. 

Yes, potentially, but IMO, comma delimited CIDR addresses would be the correct approach to take if I were voting on this feature.

Engineering priorities aren't always as simple as slapping on direct feature requests willy nilly. It's the intent of the request that matters. 

And while I don't work at Coinbase/GDAX, I know that certain architectural decisions sometimes complicate what sounds like a simple feature add. Not to mention, I think Coinbase/GDAX has other much more pressing priorities at the moment -- especially related to the stability of the exchange as a whole. 

In summary, I'm in agreement with you about the nicety of your feature request; I agree it would be ideal, but in the meantime, multiple API keys is a workaround that you can fairly easily program against and accomplish your underlying goals today.
